### ISSUE TITLE
Textbelt suppresses texts with URLs

### ISSUE THREAD

**Issue by alexpgates** [rogeruiz/cvs-covid-vaccines-newyork#2]
I wrote a little test for Textbelt to run on my server, as I was a bit paranoid about catching an opening but having the SMS fail.

It turns out that Textbelt suppressed my test message that contained a URL. I filled out a request form to allow a particular URL.. but maybe it's best to remove the URL from the alert message by default, as it's possible that an opening could be found but the alert message gets suppressed.

**Comment by rogeruiz** [comment]
I had the same issue. They opened it up for me to add URLs after I emailed them about it. I think the README should be updated and perhaps we make the URL optional by making it an external environment variable? Thanks!

**Comment by alexpgates** [comment]
Yes, I found it was easy to get this enabled after emailing as well. Dig the idea of using env variables for this and updating the README. I'll familiarize myself with how to do this in BASH and will submit a PR for both changes. Thank you!
